Configuration/AlCa/python/autoCond.py

```diff
@@ -44,6 +44,8 @@
     # GlobalTag for Run2 HLT for HI (not 2018 HI): it points to the online GT
     'run2_hlt_hi'       :   '101X_dataRun2_HLTHI_frozen_v9',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2017 (and 0,0,~0-centred beamspot)
+    'run3_data'         :   '106X_dataRun3_Express_Candidate_2019_05_24_21_36_45',
```

Comment:
@boudoul in the past 'run2_data' key was reserved for offline / re-reco Global Tags. I think to preserve the nomenclature something like `run3_data_promptlike` would be more appropriate. By the way do we have a Run3 prompt queue?

Reply:
I agree that `run3_data` should be reserved for offline global tags. 

@boudoul Do we plan to have prompt reco for the MWGRs or only express? We do have both express and prompt queues for Run 3: "106X_dataRun3_Express_Queue" and "106X_dataRun3_Prompt_Queue". But if we are not planning to have prompt reco in the near future, we should have an express GT here: there's no point in validating a GT that we don't plan to use. I would be inclined to use the key name `run3_data_promptlike` in either case.